flink-clients/src/main/java/org/apache/flink/client/program/PackagedProgram.java

```diff
@@ -516,7 +590,7 @@ private static void checkJarFile(URL jarfile) throws ProgramInvocationException
 
 		private List<URL> userClassPaths = Collections.emptyList();
 
-		private Configuration configuration = new Configuration();
+		private Configuration configuration = null;
```

Comment:
Why don't we set a default, empty `Configuration` anymore? Like this we need to add a lot of calls in tests that just set an empty `Configuration`.

Reply:
fixed